Makefile

```diff
@@ -54,23 +54,17 @@ FAIL_COVERAGE = { echo '$(COLOUR_RED)FAIL - Coverage below $(COVERAGE)%$(COLOUR_
 
 # --- Lint ---------------------------------------------------------------------
 GOLINT_VERSION = 1.33.2
-GOLINT_INSTALLED_VERSION = $(or $(word 4,$(shell golangci-lint --version 2>/dev/null)),0.0.0)
-GOLINT_MIN_VERSION = $(shell printf '%s\n' $(GOLINT_VERSION) $(GOLINT_INSTALLED_VERSION) | sort -V | head -n 1)
-GOPATH1 = $(firstword $(subst :, ,$(GOPATH)))
-LINT_TARGET = $(if $(filter $(GOLINT_MIN_VERSION),$(GOLINT_VERSION)),lint-with-local,lint-with-docker)
+GOBIN ?= $(firstword $(subst :, ,$(GOPATH)))/bin
+GOLANGCI_LINT = golangci-lint-v$(GOLINT_VERSION)
```

Comment:
for consistency with the version variable I suggest you rename this to `GOLINT` (and fix the references below)